beetsplug/discogs.py

```diff
@@ -197,6 +197,32 @@ def candidates(self, items, artist, album, va_likely, extra_tags=None):
             self._log.debug('Connection error in album search', exc_info=True)
             return []
 
+    @staticmethod
+    def extract_release_id_regex(album_id):
```

Comment:
Probably OK to just call this `extract_release_id`.

Reply:
There is already a method called `extract_release_id`, it  caused some confusion when testing